src/queries/get-10-days-share-value.graphql

```diff
@@ -0,0 +1,13 @@
+query Get10DaysShareValue($epoch: Int!, $cellarAddress: ID!) {
+  cellar(id: $cellarAddress) {
+    dayDatas(
+      first: 10
```

Comment:
Would it not be easier to send `epoch = Math.floor(Date.now() / 1000)` and order `desc`

Reply:
Then you don't need to calculate the previous 10 days.